### ISSUE TITLE
get user allocation id protoflux node

### ISSUE THREAD

**Issue by art0007i** [Yellow-Dog-Man/Resonite-Issues#681]
### Is your feature request related to a problem? Please describe.

I saw the protoflux impulse displays have a bar which is colored according to the users allocation id. I wanted to do something similar in a custom creation.

### Describe the solution you'd like

A protoflux node with a user input and byte output that gives the allocation id.

### Describe alternatives you've considered

forcing users to duplicate a slot and then parsing that slot's refid

**Comment by Frooxius** [comment]
What is the use case for this? I'm not sure if this is something we'd want to provide right now, because it's not likely to remain stable/same in the future, which would result in breakage.

The AllocationID is more of an internal value and you should not rely on it being in particular style format. It'll likely change away from byte in the future, which would break this.

I'd strongly recommend not parsing it out of RefID's either. That will break at some point.

**Comment by art0007i** [comment]
I have a table that displays users and I want it to be ordered by allocation id, but the table entries are all generated by host,

if it's planned to not be a byte in the future then the node could cast to an int or any other number type.

**Comment by MrVoltz** [comment]
I agree with Frooxius here, a simple solution would be to calculate some hash of the user's username (for example a sum of the character codes, or some polynomial hash if you want to be fancy).

**Comment by Frooxius** [comment]
We don't know if it's going to be castable to int in the first place, it might end up having more structure than a single number. The main point is that we don't want to commit to specific representation at this point.

Why do you need to sort by Allocation ID? This is not really property that users should worry about at such low level.

**Comment by art0007i** [comment]
because It's a property that I want to see, but if allocation id is gonna get changed/removed then this request can be disregarded
